spark_client/cli/service_account_registry.py

```diff
@@ -126,7 +126,7 @@ def create_service_account_registry_parser(parser: ArgumentParser):
 
     elif args.action == Actions.DELETE:
         user_id = build_service_account_from_args(args).id
-        logging.info(user_id)
+        logging.critical(user_id)
```

Comment:
**question** why critical? Info seems fine. This does not seem something very critical to log out

Reply:
Actually we absolutely need to print this to the console but are using the logging interface for it.

Since the default log level is ERROR, this will never get printed out to console unless we use log level ERROR or CRITICAL.

Earlier we were using bare print statements instead of the logging interface.

Reply:
Discussed with @welpaolo, we should substitute the logging in "cli" files with "print", possibly adding some useful messages